src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIClientExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static IChatClient AsIChatClient(this OpenAIResponseClient responseClient
     /// <param name="audioClient">The client.</param>
     /// <returns>An <see cref="ISpeechToTextClient"/> that can be used to transcribe audio via the <see cref="AudioClient"/>.</returns>
     [Experimental("MEAI001")]
-    public static ISpeechToTextClient AsSpeechToTextClient(this AudioClient audioClient) =>
+    public static ISpeechToTextClient AsISpeechToTextClient(this AudioClient audioClient) =>
```

Comment:
Went didn't any tests need to change?

Reply:
https://github.com/dotnet/extensions/pull/5838/commits/dd5ec14e65724be5e813d856a1dda1fee3d75855

Added some missing UT, in line with what we have for Chat.